internal/drain-config/drainConfig.go

```diff
@@ -45,7 +45,10 @@ func NewDrainConfigManager(conf *config.Config, log logger.Logger) (*drainConfig
 		log.Errorw("db setup", "error", err)
 		return nil, fmt.Errorf("db setup: %v", err)
 	}
-	if err := migrate(db); err != nil {
+	if err := db.Ping(); err != nil {
```

Comment:
We shouldn't ignore errors in case we are in processor, so whether it will fail or not, should be configurable